cmd/abapEnvironmentCloneGitRepo.go

```diff
@@ -63,9 +64,9 @@ func runAbapEnvironmentCloneGitRepo(config *abapEnvironmentCloneGitRepoOptions,
 		Password:           connectionDetails.Password,
 	})
 
-	repositories, errGetRepos := getRepositories(config)
+	repositories, errGetRepos := abaputils.GetRepositories(&abaputils.RepositoriesConfig{BranchName: config.BranchName, RepositoryName: config.RepositoryName, Repositories: config.Repositories})
 	if errGetRepos != nil {
-		return errGetRepos
+		return fmt.Errorf("Something failed during the clone: %w", errGetRepos)
```

Comment:
I don't think it makes sense to wrap the error message at this point. "Something failed during the clone" does not provide valuable information and only makes it harder to read the error message.